lib/onlyoffice_webdriver_wrapper/webdriver/webdriver_screenshot_helper.rb

```diff
@@ -4,8 +4,10 @@ module WebdriverScreenshotHelper
     def get_screenshot_and_upload(path_to_screenshot = "/mnt/data_share/screenshot/WebdriverError/#{StringHelper.generate_random_string}.png")
       begin
         get_screenshot(path_to_screenshot)
-        path_to_screenshot = AmazonS3Wrapper.new.upload_file_and_make_public(path_to_screenshot, 'screenshots')
-        OnlyofficeLoggerHelper.log("upload screenshot: #{path_to_screenshot}")
+        cloud_screenshot = AmazonS3Wrapper.new.upload_file_and_make_public(path_to_screenshot, 'screenshots')
+        File.delete(path_to_screenshot)
```

Comment:
may be it would be useful to have here extra check for file deletion: i.e. ...File.exist? path_to_screenshot ....